sgt-back-end/index.js

```diff
@@ -0,0 +1,124 @@
+const fs = require('fs');
+const express = require('express');
+const app = express();
+app.use(express.json());
+const pg = require('pg');
+
+// only create ONE pool for your whole server
+const db = new pg.Pool({
+  connectionString: 'postgres://dev:dev@localhost/studentGradeTable',
+  ssl: {
+    rejectUnauthorized: false
+  }
+});
+
+app.get('/api/grades', (req, res, next) => {
+  const sql = `
+    select *
+    from "grades"
+    `;
+  db.query(sql, [])
```

Comment:
If there aren't any SQL parameters you don't need to pass anything as the second argument to `db.query`. For example using `db.query(sql)` here would be fine.